src/server/auth/server/testing/oidc_test.go

```diff
@@ -4,143 +4,29 @@ import (
 	"context"
 	"net/http"
 	"net/url"
-	"strings"
 	"testing"
 
 	"golang.org/x/oauth2"
 
-	"github.com/pachyderm/pachyderm/src/client"
 	"github.com/pachyderm/pachyderm/src/client/auth"
-	"github.com/pachyderm/pachyderm/src/client/identity"
 	"github.com/pachyderm/pachyderm/src/client/pkg/require"
-	"github.com/pachyderm/pachyderm/src/server/pkg/backoff"
 	tu "github.com/pachyderm/pachyderm/src/server/pkg/testutil"
 )
 
-// Dex's mock connector always returns the same identity for all authentication
```

Comment:
Refactored this code into testutils so we can enable OIDC auth for the CLI tests as well.